website/docs/r/git_repository.html.markdown

```diff
@@ -68,9 +84,11 @@ The following arguments are supported:
 
 `initialization` - (Required) block supports the following:
 
-- `init_type` - (Required) The type of repository to create. Valid values: `Uninitialized`, `Clean` or `Import`. Defaults to `Uninitialized`.
+- `init_type` - (Required) The type of repository to create. Valid values: `Uninitialized`, `Clean` or `Import`.
 - `source_type` - (Optional) Type of the source repository. Used if the `init_type` is `Import`. Valid values: `Git`.
 - `source_url` - (Optional) The URL of the source repository. Used if the `init_type` is `Import`.
+- `username` - (Optional) The username to use when importing a private git repository.
```

Comment:
the username and password is incorrect; should be `service_connection_id`: will wait for other feedback before fixing to avoid multiple builds